test_util/test_aws_cf.py

```diff
@@ -104,7 +99,8 @@ def check_environment():
 def main():
     options = check_environment()
     cf, ssh_info = provide_cluster(options)
-    cluster = test_util.cluster.Cluster.from_cloudformation(cf, ssh_info, load_string(options.ssh_key_path))
+    # hidden hook where user can supply an ssh_key for a preexisting cluster
+    cluster = test_util.cluster.Cluster.from_cloudformation(cf, ssh_info, load_string('ssh_key'))
```

Comment:
This hack is unbearably ugly. Will fully remove the preexisting cluster pathway like I did in the other test executor files